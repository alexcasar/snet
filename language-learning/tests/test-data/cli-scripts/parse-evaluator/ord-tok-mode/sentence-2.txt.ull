jack can ['] [t] sleep at night .
0 ###LEFT-WALL### 1 jack
0 ###LEFT-WALL### 8 .
1 jack 5 sleep
1 jack 6 at
2 can 5 sleep
5 sleep 6 at
6 at 7 night

