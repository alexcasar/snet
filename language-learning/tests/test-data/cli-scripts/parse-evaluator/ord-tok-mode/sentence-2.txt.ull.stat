Parse quality:	  0.00%

Average total links:	4.00
Average ignored links:	2.00
Average missing links:	4.00
Average extra links:	5.00

Recall:		0.0000
Precision:	0.0000
F1:		0.0000

Total sentences:	1.00
Skipped sentences:	0.00

