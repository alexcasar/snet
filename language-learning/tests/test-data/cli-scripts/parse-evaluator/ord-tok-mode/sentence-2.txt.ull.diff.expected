Sentence appear to have different tokenization:
jack can't sleep at night .
in tokens:["can't"]<--->["'", 'can', 't']

